packages/fluentui/docs/src/examples/components/Menu/Visual/MenuExampleCompose.shorthand.tsx

```diff
@@ -19,32 +23,80 @@ const MenuItemIconGreen = compose(MenuItemIcon, {
   displayName: 'MenuItemIconGreen',
 });
 
-const MenuItemContentPink = compose(MenuItemContent, {
-  displayName: 'MenuItemContentPink',
+const MenuItemContentPurple = compose(MenuItemContent, {
+  displayName: 'MenuItemContentPurple',
 });
 
 const MenuitemIndicatorSaturated = compose(MenuItemIndicator, {
   displayName: 'MenuitemIndicatorSaturated',
 });
 
-const MenuItemBlue = compose(MenuItem, {
-  displayName: 'MenuItemBlue',
+interface MenuItemBlueOwnProps {
+  level?: number;
+}
+
+interface MenuItemBlueProps extends MenuItemProps, MenuItemBlueOwnProps {}
+
+const MenuItemBlue = compose<'a', MenuItemBlueProps, MenuItemStylesProps, MenuItemProps, MenuItemStylesProps>(
+  MenuItem,
+  {
+    displayName: 'MenuItemBlue',
+    slots: {
+      icon: MenuItemIconGreen,
+      content: MenuItemContentPurple,
+      indicator: MenuitemIndicatorSaturated,
+      wrapper: MenuItemWrapperDashed,
+    },
+    mapPropsToSlotProps: (props: MenuItemProps & { level?: number }) => ({
```

Comment:
```suggestion
    mapPropsToSlotProps: (props) => ({
```

This should work after #13262 will be merged

Reply:
@assuncaocharles can you make sure to remove the typings in this file?